### DISCUSSION TITLE
Remove destination account from transaction via rule

### DISCUSSION THREAD

**Discussion by NullEnt1ty** [firefly-iii/firefly-iii#7779] [Firefly III] [answered]
Hey! 🙂 

Is it possible to create a rule that removes the destination account from a transaction? Sometimes I withdraw cash and when I import my transactions into Firefly the destination account is set to the bank where I withdrew the money. This is expected, because it looks like a "normal" transaction to Firefly.

However I want to track it as a cash withdrawal in Firefly. Ideally using a rule so that the transactions are automatically corrected next time I import them from my bank account.

I already tried to use the action "Set destination account to ..." but this does not accept an empty value.

**Comment by JC5** [comment, marked as answer]
That's currently not possible, but a good idea to add!

**Comment by diamant-x** [comment]
I normally handle cash withdrawals through a rule that converts the withdrawal to a transfer to a 'Cash Wallet' asset account (with source account my bank account).
Then is up to you to track where do you spend cash on.
I try to be as rigorous as it makes sense, but once per month I just check my remaining cash in my wallet, check what my firefly 'cash wallet' remaining balance is, and create a 'fake' withdrawal to reflect the difference (Which normally should be from adding several small purchases).

PS: In your case, if you don't want to track at all any of your cash expenses, then it would be 1) rule for the trasnfer from bank to cash wallet account, and 2) rule for when a new 'transfer transaction' into cash wallet account occurs, a new withdrawal for the total amount of balance in cash account is created with empty destination account. (not sure if rules can create new transactions, when another transaction)
